tests/fixtures/testrepo/downstream/package_manifest.yaml

```diff
@@ -1,12 +1,15 @@
 ---
 packages:
   vars:
-    diff_brew_package_skip: false
+    diff_package_skip: false
+    scl: tfm
     build_package_koji_command: 'brew'
     satellite_version: '6.3.0'
     foremandist: 'fm1_15'
     releasers:
       - obaltest-dist-git-rhel-7
+    brew_tags:
+      - satellite-6.3.0-rhel-7-candidate
```

Comment:
Should we rename this to `obaltest-rhel-7-candidate`? Just so that we don't have `satellite` in here?